README.md

```diff
@@ -79,7 +79,7 @@ This package includes all source columns defined in the macros folder. You can a
 
 vars:
   qualtrics_source:
-    survey_pass_through_columns:
+    survey_pass_through_columns: 
```

Comment:
Could we adjust these variables to be consistent with all other variables in the package. So for these just adding `qualtrics__` before the variable name. Or do you have an argument against that adjustment?

Reply:
ah no i agree! that way users for sure won't need to add the variables twice (scoped under `qualtrics_source` and `qualtrics`)